jena-cmds/src/main/java/riotcmd/utf8.java

```diff
@@ -60,7 +60,7 @@ public static void main(String... args) {
                     } else
                         colNum++ ;
                     if ( !Character.isDefined(ch) )
-                        throw new AtlasException(String.format("No such codepoint: 0x%04X", ch)) ;
+                        System.out.printf("No such codepoint: 0x%04X\n", ch);
```

Comment:
Any reason why this is now just a log message rather than an error as it was previously?

Reply:
Previous there was also a Character.isDefined in InStreamUTF8 and also a bad test that excluded surrogate use.

It is like this so first error does not terminate the command. Of course, it may flip to many errors!

When trying it out (on the rdf-tests data which is not large), stopping on the first error was unhelpful because the errors were in several bytes (surrogate pairs by the time it got to here).

Reply:
Makes perfect sense